doc/api/errors.md

```diff
@@ -563,6 +570,67 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_INVALID_ARG_TYPE"></a>
+### ERR_INVALID_ARG_TYPE
+
+The `'ERR_INVALID_ARG_TYPE'` error code is used generically to identify that
+an argument of the wrong type has been passed to a Node.js API.
+
+<a id="ERR_INVALID_CALLBACK"></a>
+### ERR_INVALID_CALLBACK
+
+The `'ERR_INVALID_CALLBACK'` error code is used generically to identify that
+a callback function is required and has not been provided to a Node.js API.
+
+<a id="ERR_INVALID_REPL_EVAL_OPTION"></a>
+### ERR_INVALID_REPL_EVAL_OPTION
+
+The `'ERR_INVALID_REPL_EVAL_OPTION'` error code is used when both
+`breakEvalOnSigint` and `eval` options are set in the REPL config.
+Allowing both of these would be confusing as `breakEvalOnSigint`
```

Comment:
I was trying to explain the rationale of it not being supported, but you are right, it is not the place to do it. I changed the formulation.